### ISSUE TITLE
Node crashed:  "fatal runtime error: failed to initiate panic, error 5"

### ISSUE THREAD

**Issue by frozentoez** [input-output-hk/shelley-testnet#105]
Running versions: 
-----------------
jcli 0.5.6 (HEAD-7ab929e+, release, linux [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)]
jormungandr 0.5.6 (HEAD-7ab929e+, release, linux [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)]

Operating System:
-----------------
NAME="Ubuntu"
VERSION="18.04.3 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.3 LTS"
VERSION_ID="18.04"

Installer:
----------
https://github.com/input-output-hk/jormungandr/releases/download/v0.5.6/jormungandr-v0.5.6-x86_64-unknown-linux-gnu.tar.gz

Running config:
---------------
log:
  format: "plain"
  level: "info"
  output: "stderr"
p2p:
  listen_address: "/ip4/0.0.0.0/tcp/3100"
  public_address: "/ip4/{my_public_ip_here}/tcp/3100"
  trusted_peers:
    - "/ip4/3.123.177.192/tcp/3000"
    - "/ip4/3.123.155.47/tcp/3000"
    - "/ip4/52.57.157.167/tcp/3000"
    - "/ip4/3.112.185.217/tcp/3000"
    - "/ip4/18.140.134.230/tcp/3000"
    - "/ip4/18.139.40.4/tcp/3000"
    - "/ip4/3.115.57.216/tcp/3000"
  topics_of_interest:
    messages: "high"
    blocks: "high"
rest:
  listen: "127.0.0.1:3101"
storage: "/home/shelley/testnet/storage"

Node stats run just 6 minutes prior to error below:
---------------------------------------------------
blockRecvCnt: 661
lastBlockDate: "228.34503"
lastBlockFees: 0
lastBlockHash: 94a2e1080211435920bd6dabae40598e6b732b93b10b565e7c037915837a8048
lastBlockHeight: "77373"
lastBlockSum: 0
lastBlockTime: "2019-10-09T04:06:09+00:00"
lastBlockTx: 0
txRecvCnt: 0
uptime: 10040

Error message and node crashed:
-------------------------------
Oct 09 04:12:04.474 INFO client P2P connection closed, node_id: 7639909459871603480, peer_addr: 165.227.201.249:3000, task: network
thread '<unnamed>' panicked at 'failed to allocate an alternative stack', src/libstd/sys/unix/stack_overflow.rs:134:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
thread '<unnamed>' panicked at 'failed to allocate an alternative stack', src/libstd/sys/unix/stack_overflow.rs:134:13
thread '<unnamed>' panicked at 'failed to allocate an alternative stack', src/libstd/sys/unix/stack_overflow.rs:134:13
thread '<unnamed>' panicked at 'failed to allocate an alternative stack', src/libstd/sys/unix/stack_overflow.rs:134:13
fatal runtime error: failed to initiate panic, error 5

**Comment by dorin100** [comment]
Hi @frozentoez . Can you please provide all the node logs? 

For the future, it would help us if you would execute `set RUST_BACKTRACE=full` before starting the node. This would collect more details about the node crash.

**Comment by yuta-taka** [comment]
I have same crash: "fatal runtime error: failed to initiate panic, error 5"
so i report it here.

★Prerequisites

・Running versions:Operating System:Installer:→Same

・Running config:
log:
  format: "plain"
  level: "info"
  output: "stderr"
p2p:
  listen_address: "/ip4/0.0.0.0/tcp/9000"
  public_address: "/ip4/13.58.234.3/tcp/9000"
  topics_of_interest:
    blocks: "high"
    messages: "high"
  trusted_peers:
    - "/ip4/3.123.177.192/tcp/3000"
    - "/ip4/3.123.155.47/tcp/3000"
    - "/ip4/52.57.157.167/tcp/3000"
    - "/ip4/3.112.185.217/tcp/3000"
    - "/ip4/18.140.134.230/tcp/3000"
    - "/ip4/18.139.40.4/tcp/3000"
    - "/ip4/3.115.57.216/tcp/3000"
rest:
  listen: "127.0.0.1:3101"
storage: "~/"

・Node stats run just 6 minutes prior to error below:&Error message and node crashed:→below
[failedtoinitiate panicerror 5nohup.out.txt](https://github.com/input-output-hk/shelley-testnet/files/3710077/failedtoinitiate.panicerror.5nohup.out.txt)

**Comment by dorin100** [comment]
Thank you @yuta-taka . I will log a separate issue in order for our developers to check the provided logs.

Please execute this command - https://github.com/input-output-hk/shelley-testnet/issues/105#issuecomment-539872526 - before starting the node from now on. It will provide more details in the case of node crashes.

**Comment by dorin100** [comment]
The issue was moved to the main development repo - https://github.com/input-output-hk/jormungandr/issues/924
